src/main/java/io/quarkus/security/identity/SecurityIdentity.java

```diff
@@ -133,4 +139,30 @@ default <T extends Principal> T getPrincipal(Class<T> clazz) {
     default boolean checkPermissionBlocking(Permission permission) {
         return checkPermission(permission).await().indefinitely();
     }
+
+    /**
+     * Checks if a user holds a given permission.
+     * <p>
+     * This method is asynchronous, as it may involve calls to a remote resource.
+     *
+     * @param permission The permission
+     * @return A completion stage that will resolve to true if the user has the specified permission
+     */
+    default Uni<Boolean> checkPermission(String permission) {
```

Comment:
These are just shortcuts for the already existing `checkPermission(Permission)` so I guess they are OK too ?

Reply:
yes that is ok

Reply:
and nice (but I thought that goes without saying)